### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

File: example.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 'use strict'
```
Comment (diasdavid) [inline example.js]: Could you please move this example to the examples folder? I just realised now that it ended up in the root path

File: example.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
+// print the node id
 
 // IPFS_PATH will point to /tmp/ipfs_***** and will be
 // cleaned up when the process exits.
 
-const ipfsd = require('ipfsd-ctl')
+const ipfsd = require('.')
```
Comment (diasdavid) [inline example.js]: Does this work? It should have to point to `src`

File: examples/id.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-var ipfsd = require('../index.js')
+var ipfsd = require('..')
```
Comment (diasdavid) [inline examples/id.js]: same as above

File: examples/local.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-var ipfsd = require('../index.js')
+var ipfsd = require('..')
```
Comment (diasdavid) [inline examples/local.js]: same as above

File: example.js (hunk 2 above)
Comment (thisconnect) [inline example.js]: Yes this works, though in the Node.js docs `require('./')` with a slash is commonly used. As it is pointing to the root of this module Node.js will lookup the main field in package.json and so it finds `src/index.js`. 
https://nodejs.org/api/modules.html#modules_all_together

If the examples point to the root of the module, the src files and main field in package.json could be refactored and moved without changing the examples. But I'm happy to change the examples/* to point to `../src` if you want.

**Comment by diasdavid** [comment]
Thank you @thisconnect :)

**Pull request merged**